packages/side-nav/src/SideNav.spec.tsx

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render, cleanup } from '@testing-library/react';
+import { MenuItemProps } from '@leafygreen-ui/menu';
+import { SideNav, SideNavGroup, SideNavItem } from './index';
+
+type renderedElement = HTMLElement | null | undefined;
+
+interface RenderedElements {
+  navEl?: renderedElement;
+  groupEl?: renderedElement;
+  headerEl?: renderedElement;
+  itemEl?: renderedElement;
+  linkEl?: renderedElement;
+}
+
+afterAll(cleanup);
+
+describe('packages/side-nav', () => {
+  const className = 'test-class-name';
+  const headerText = 'test-header-text';
+  const headerClassName = 'test-header-class-name';
+  const testIds = {
+    sideNav: 'side-nav',
+    sideNavGroup: 'side-nav-group',
+    sideNavItem: 'side-nav-item',
+    sideNavLink: 'side-nav-link',
+  };
+
+  let renderedEls: RenderedElements = {};
+
+  afterEach(() => {
+    document.body.innerHTML = '';
+    renderedEls = {};
```

Comment:
[nitpick] I can remove the `afterAll` and add a call to `cleanup` here since this `afterEach` is at the root.